src/hooks/useOwnedSafes.ts

```diff
@@ -16,26 +16,23 @@ type OwnedSafesCache = {
 }
 
 const useOwnedSafes = (): OwnedSafesCache['walletAddress'] => {
-  const { safe } = useSafeInfo()
+  const chainId = useChainId()
   const { address: walletAddress } = useWallet() || {}
   const [ownedSafesCache, setOwnedSafesCache] = useLocalStorage<OwnedSafesCache>(CACHE_KEY)
 
-  const [ownedSafes, error] = useAsync<OwnedSafes>(() => {
-    if (!safe.chainId || !walletAddress) return
-    return getOwnedSafes(safe.chainId, walletAddress)
-  }, [safe.chainId, walletAddress])
+  const [_, error] = useAsync<void>(async () => {
```

Comment:
Syntax tip:
```suggestion
  const [, error] = useAsync<void>(async () => {
```
(Irrelevant tho, see my other comment).